src/lsp/cobol_lsp/lsp_data_info_printer.ml

```diff
@@ -25,16 +25,37 @@ let pp_cobol_block: _ Fmt.t -> _ Fmt.t = fun pp ->
 
 (* usage *)
 
+let max_value digits scale =
+  let s = "123456789123456789123456789123456789" in
+  let whole = (digits - scale) in
+  let scale = if scale < 0 then 0 else scale in
+  let whole_part = Str.string_before s whole in
+  let decimal_part = Str.string_before (Str.string_after s whole) scale in
+  float_of_string (whole_part ^ "." ^ decimal_part)
+
+let pp_example_of ppf (picture: Cobol_data.Picture.t) =
+  match picture.category with
+  | FixedNum { digits; scale; _ } ->
+      let max = max_value digits scale in
+      let max_str =
+      if Float.is_integer max
+      then string_of_int (int_of_float max)
+      else string_of_float max in
+    Fmt.pf ppf "\n\nExample display value:\n- 0: `%s`\n- %s: `%s`"
```

Comment:
Isn't there a better formatting we could find for this, more on a single line, simply separated with commas? Also `Example display value**s**`, as a heading and without `:`.
Next, I'd put the actual rendering first, and then the actual raw value that is rendered in between parentheses.